_includes/release-assets.html

```diff
@@ -31,12 +31,19 @@
 </td>
 <td>
 	<ul class="row">
+		{% assign signed_firefox = include.assets | where_exp:"item", "item.name contains '-firefox.'" | first %}
+		{% assign unsigned_firefox = include.assets | where_exp:"item", "item.name contains '-firefox-unsigned'" | first %}
 
-		{% assign firefox = include.assets | where_exp:"item", "item.name contains '-firefox.'" | first %}
-		{% if firefox != nil %}
-			<li><a href="{{ firefox.browser_download_url}}" target="_blank">Firefox</a></li>
+		{% if signed_firefox != nil %}
+		<li><a href="{{ signed_firefox.browser_download_url}}" target="_blank">Firefox</a></li>
+		{% elsif include.latest == nil and unsigned_firefox != nil %}
```

Comment:
Can't `include.latest == nil` be `not include.latest`? Quite hypothetic, but what if someone passes `latest = false`?

Reply:
https://github.com/Shopify/liquid/issues/138#issuecomment-657514982

Reply:
I could just pass latest=false explicitly, but then there's the opposite problem of that always having to be done. Your choice.

Reply:
Maybe it could be `include.latest != true`?